src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Rand.txt

```diff
@@ -59,4 +59,19 @@ Error({Kind:ErrorKind.Numeric})
 400
 
 >> Find("000000000000", Text(RandBetween(-1e28, 1e28)))
-Blank()
\ No newline at end of file
+Blank()
+
+>> RandBetween( 9007199254740992 /* MaxFloatInt */, 9007199254740992 /* MaxFloatInt */ )
+9007199254740992
+
+>> RandBetween( -9007199254740992 /* MaxFloatInt */, -9007199254740992 /* MaxFloatInt */ )
+-9007199254740992
+
+>> Sum(Sequence(100), With( { r: RandBetween( 9007199254740985, 9007199254740992 /* MaxFloatInt */ ) }, If( r >= 9007199254740985 And r <= 9007199254740992, 1, Error( "Result out of range" ) ) ) )
```

Comment:
>r >= 9007199254740985 And r <= 9007199254740992 [](http://example.com/codeflow?start=108&length=47)

Suggestion: `9007199254740985 <= r And r <= 9007199254740992`: easier to understand the boundaries

Reply:
Done.